static_src/dispatcher.js

```diff
@@ -0,0 +1,19 @@
+
+import {Dispatcher} from 'flux';
+
+class AppDispatcher extends Dispatcher {
+  handleViewAction(action) {
+    action.source = 'VIEW_ACTION';
```

Comment:
Right now they're more for debugging purposes. They're also there if the system gets more complex, the stores would potentially use them, but I'm not thinking of that right now.

For now, I like having them in tests because it's being very explicit about where actions are coming from, making the tests more descriptive (and a little easier to make them fail).